mainframer.sh

```diff
@@ -40,6 +40,10 @@ mkdir -p "$PROJECT_DIR"/build
 # Remove previous archives of the project.
 rm -f "$PROJECT_DIR"/build/project_for_remote_build.tar "$PROJECT_DIR"/build/remotely_built_project.tar
 
+# Archiver.
+
+ARCHIVER="$(eval "if type 'pigz' > /dev/null; then echo 'pigz'; else echo 'gzip'; fi")"
```

Comment:
You're evaling it right now, make it lazy so it'll work correctly on remote machine!